src/components/FilterSort.vue

```diff
@@ -0,0 +1,233 @@
+          >
+            <input
+              id="submitted"
+              v-model="filtersSortData.status"
+              class="form-check-input"
+              type="checkbox"
+              :value="status.value"
+              @change="emitFilterSortData"
+            >
+            <label
+              class="form-check-label text-sort-filter-dropdown-text"
+              for="submitted"
+            >
+              {{ status.label }}
+            </label>
+          </div>
+        </ul>
+      </div>
+
+      <div class="filter-by-date">
+        <button
+          class="btn btn-sm btn-outline-sort-filter-button-outline dropdown-toggle"
+          :class="filtersSortData.dateStart !== '' || filtersSortData.dateEnd !== '' ? 'btn-primary show':''"
+
+          type="button"
+          data-bs-toggle="dropdown"
+          aria-expanded="false"
+        >
+          Filter by date
+        </button>
+        <ul
+          class="dropdown-menu"
+          aria-labelledby="dropdownMenuButton1"
+        >
+          <div class="mx-2 my-2 dropdown-contents text-sort-filter-dropdown-text">
+            <div class="d-flex align-items-center mb-2">
+              <label
+                class="pe-2 w-25"
+                for="startDate"
+              >Start:</label>
+              <input
+                id="startDate"
+                v-model="filtersSortData.dateStart"
+                class="form-control form-control-sm text-sort-filter-dropdown-text"
+                type="date"
+                @input="emitFilterSortData"
+              >
+            </div>
+            <div class="d-flex align-items-center">
+              <label
+                for="endDate"
+                class="pe-3 w-25"
+              >End:</label>
+              <input
+                id="endDate"
+                v-model="filtersSortData.dateEnd"
+                class="form-control form-control-sm text-sort-filter-dropdown-text"
+                type="date"
+                @input="emitFilterSortData"
+              >
+            </div>
+          </div>
+        </ul>
+      </div>
+    </div>
+
+    <div class="my-2 ms-auto">
+      <button
+        type="button"
+        class="btn btn-sm btn-outline-sort-filter-clear-button-outline"
+        @click="clearAllFilters()"
+      >
+        <i class="bi bi-x-circle" />
+
+        Clear all filters
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+import moment from "moment"
+import { ROLES } from "@/config/consts"
+
+export default {
+  name: "NegotiationsList",
+  props: {
+    filtersSortData: {
+      type: Object,
+      default: undefined
+    },
+    userRole: {
+      type: String,
+      required: true,
+      validator: function (value) {
+        return [ROLES.RESEARCHER, ROLES.REPRESENTATIVE].includes(value)
+      }
+    }
+  },
+  emits: ["filtersSortData"],
+  data () {
+    return {
+      filtersStatus: [
+        { value: "SUBMITTED", label: "Submitted" },
```

Comment:
the label in Negotiation list is "UNDER REVIEW". Here it should be the same otherwise it may be misleading